batch7crud-roll4/src/main/java/com/lftechnology/batch7crud/util/DbConnection.java

```diff
@@ -0,0 +1,30 @@
+package com.lftechnology.batch7crud.util;
+
+import com.lftechnology.batch7crud.exception.DataException;
+
+import javax.naming.Context;
+import javax.naming.InitialContext;
+import javax.naming.NamingException;
+import javax.sql.DataSource;
+import java.sql.*;
+
+/**
+ * Created by pratishshr on 1/14/16.
+ */
+public class DbConnection {
+
+    public static Connection getConnection() throws DataException {
+        try {
+            Context initCtx = new InitialContext();
+            Context envCtx = (Context) initCtx.lookup("java:comp/env");
+            DataSource ds = (DataSource) envCtx.lookup("jdbc/ems");
+
+            Connection connection = ds.getConnection();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "connection". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1488)